libr/core/panels.c

```diff
@@ -4193,10 +4193,16 @@ static RPanels *get_cur_panels(RPanelsRoot *panels_root) {
 }
 
 static bool handle_tab(RCore *core) {
+	char msg[128];
```

Comment:
dont use static buffers. use r_str_newf or so, but if u want static buffer, make it of the screen width, not a fixed 128 length

Reply:
ah okay hang on